clang build error

**Operating system**: Windows 10 64bit/MSYS2/mingw-w64/CLANG64 
**wxPython version & source**:          self-built wxPython-4.1.2a1.dev5330+3e6be81d
                                                            custom MSys2 pkgbuild build using sip4 and waf
**Python version & source**:              MSys2 distro 3.9.12
clang version 14.0.3
sip 4.19.25
waf: 2.0.20

**Description of the problem**: The clang compiler gives build error.

```
../../sip/cpp/sip_corewxMenuItem.cpp:41:20: error: taking the address of a temporary object of type 'wxBitmap' [-Waddress-of-temporary]
            return &self->GetDisabledBitmap();
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~
```
[Link to possible fix: ](https://github.com/msys2/MINGW-packages/pull/11655/commits/e9581a5a589069ee3b2df6ff6072aab39a905514
)

Possible fix also added below
```
diff --git a/etg/menuitem.py b/etg/menuitem.py
index ac6cdb38..ba794039 100644
--- a/etg/menuitem.py
+++ b/etg/menuitem.py
@@ -136,11 +136,12 @@ def run():


     c.find('GetDisabledBitmap').type = 'const wxBitmap*'
+    c.find('GetDisabledBitmap').transferBack = True  # Python takes ownership of the return value
     c.find('GetDisabledBitmap').setCppCode("""\
         #ifdef __WXMSW__
-            return &self->GetDisabledBitmap();
+            return new wxBitmap(self->GetDisabledBitmap());
         #else
-            return &wxNullBitmap;
+            return new wxBitmap(wxNullBitmap);
         #endif
         """)

-- 

```

Tim S.

--- a/etg/menuitem.py
+++ b/etg/menuitem.py
@@ -136,11 +136,12 @@ def run():
 
 
     c.find('GetDisabledBitmap').type = 'const wxBitmap*'
+    c.find('GetDisabledBitmap').transferBack = True  # Python takes ownership of the return value
     c.find('GetDisabledBitmap').setCppCode("""\
         #ifdef __WXMSW__
-            return &self->GetDisabledBitmap();
+            return new wxBitmap(self->GetDisabledBitmap());
         #else
-            return &wxNullBitmap;
+            return new wxBitmap(wxNullBitmap);
         #endif
         """)
 

